haas/api.py

```diff
@@ -796,20 +796,22 @@ def port_detach_nic(switch, port):
     port.nic = None
     db.session.commit()
 
-
-@rest_call('GET', '/free_nodes', Schema({}))
-def list_free_nodes():
-    """List all nodes not in any project.
+@rest_call('GET', '/node/<is_free>', Schema({'is_free' : basestring}))
```

Comment:
I don't feel strongly about either, but for some reason list_projects is registered to `GET /projects`. https://github.com/CCI-MOC/hil/blob/master/docs/rest_api.md#list_projects so I'd prefer to be consistent. But we should discuss about this.